Description: Use autoconf $host_cpu instead of «uname -m»
 This was making the build unreproducible as the uname will return the build
 system CPU and not the host one.
Author: Guillem Jover <guillem@hadrons.org>
Origin: vendor
Forwarded: no

---
 glide3x/configure.in |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/glide3x/configure.in
+++ b/glide3x/configure.in
@@ -142,6 +142,7 @@ AM_CONDITIONAL(FX_GLIDE_CVG, test x$FX_G
 #
 # Architecture
 #
+AC_CANONICAL_HOST
 AC_ARG_ENABLE([build-architecture],
               [dnl
   --enable-build-architecture Enable AMD 3DNow instructions [default=current]],
@@ -152,7 +153,7 @@ AC_ARG_ENABLE([build-architecture],
                *)
                    AC_MSG_ERROR([Illegal value (${enableval}) for --enable-build-architecture])
                    ;;
-               esac],[FX_GLIDE_BUILD_ARCHITECTURE=`uname -m`])
+               esac],[FX_GLIDE_BUILD_ARCHITECTURE=$host_cpu])
 AC_SUBST(FX_GLIDE_BUILD_ARCHITECTURE)
 #
 # Various tests needed at points in the build
